content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,381 @@
+
+Validators play the following economic roles in AFD processing:
+
+- _loss_: loss of stake, validator reputation, staking reward revenue as the _offending_ validator of a slashed _fault_
+- _beneficiary_: gain of slashing reward as the _reporting_ validator of a slashed _fault_.
+
+Autonity community play the following economic role in AFD processing:
+
+- _beneficiary_: gain of slashed stake token for community funds.
+
+### Protocol primitives
+
+Essential primitives of AFD are: accusation, innocence, and fault proofs; slashing and severity; jailing.
+
+#### Accusations
+
+An _accusation_ is a claim that a consensus committee member has failed to participate correctly in consensus. I.e. it has failed to adhere to a rule governing validator participation in consensus and so is guilty of a rule infraction. Protocol only allows a validator to be under _accusation_ once at a time. New _accusations_ are made by submitting _proof_ of an accusable rule infraction on-chain as an event of type `NewAccusation`.
+
+Accusations do not automatically cause slashing and an _innocence proof window_ measured in blocks gives the accused _offending_ validator a window to detect an _accusation_ and prove _innocence_ by submitting an _innocence_ proof on-chain. The expiry of this window creates a _deadline_ before which a new `Accusation` proof cannot be submitted.
+
+There are protocol constraints on when an _accusation_ can be made. A _reporting_ validator can only submit an _accusation_ _proof_ if:
+
+- the _offending_ validator:
+  - does not already have a _fault_ in the epoch in which the _new accusation_ is being made for an offence with a higher _severity_
+  - is not currently already accused of committing a rule infraction, i.e. there is a _pending accusation_. In this case, a _new accusation_ cannot be made until expiry of the _innocence proof window_ during which an accused _offending_ validator is able to submit an `innocence` proof refuting the _pending accusation_. This creates a _deadline_ measured in block height before which a new `accusation` proof cannot be submitted.
+- it is within the _accusation window_:
+  - an accusation must be made `<= 256` blocks after the detected accountability event.
+
+As each block is finalised, AFD will attempt to promote _accusations_ where the _innocence proof submission window_ has expired within protocol constraints to proven _faults_.
+
+To check if an _offending_ validator has a _pending accusation_, a _reporting_ validator calls protocol functions:
+
+- [`canAccuse()`](/reference/api/accountability/#canaccuse): to determine if (a) an offending validator is accusable, and, (b) the _deadline_ number of blocks remaining in the _innocence proof submission window_ if there is a _pending accusation_.
```

Comment:
I don't know if I would list the specific functions that are called. AFAIK we do not have a standardized interface for the Accountability contract, so we might end up needing to update the docs every time there is a refactor or we change the internal logic.

I would just clearly list the conditions for a validator to be accusable and slashable, without mentioning which specific functions are called.

Reply:
Commented out in commit f0abfb26c62401bfb3d3de49897a639a5944840c.